Makefile.config

```diff
@@ -84,8 +84,10 @@ FEATURES_LIST	:= TCP_REPAIR STRLCPY STRLCAT PTRACE_PEEKSIGINFO \
 # $1 - config name
 define gen-feature-test
 ifeq ($$(call try-cc,$$(FEATURE_TEST_$(1)),$$(LIBS_FEATURES),$$(DEFINES)),true)
-	$(Q) echo '#define CONFIG_HAS_$(1)' >> $$@
+	echo '#define CONFIG_HAS_$(1)' >> $$@
 	$(Q) echo '' >> $$@
+else
+	echo '#undef CONFIG_HAS_$(1)' > /dev/null
```

Comment:
should it be something like this:
```
@@ -84,10 +84,10 @@ FEATURES_LIST       := TCP_REPAIR STRLCPY STRLCAT PTRACE_PEEKSIGINFO \
 # $1 - config name
 define gen-feature-test
 ifeq ($$(call try-cc,$$(FEATURE_TEST_$(1)),$$(LIBS_FEATURES),$$(DEFINES)),true)
-       echo '#define CONFIG_HAS_$(1)' >> $$@
+       $(E) 'CONFIG_HAS_$(1)=y'
        $(Q) echo '' >> $$@
 else
-       echo '#undef CONFIG_HAS_$(1)' > /dev/null
+       $(E) "#CONFIG_HAS_$(1) isn't set"
 endif
 endef
```

Reply:
I think not - this code generates a C header and the .config file is converted in another line in `config-header-rule` macro.